nixos/modules/misc/ids.nix

```diff
@@ -297,6 +297,7 @@
       clickhouse = 278;
       rslsync = 279;
       minio = 280;
+      nm-iodine = 281;
```

Comment:
If there is no reason to use a static uid, please consider using a dynamic uid instead.